.circleci/config.yml

```diff
@@ -255,6 +261,31 @@ jobs:
       - run:
           name: Test umd release
           command: yarn test:umd
+  test_profile:
+    <<: *defaults
+    docker:
+      - image: mcr.microsoft.com/playwright:bionic
+        environment:
+          NODE_ENV: development # Needed if playwright is in `devDependencies`
+
+    steps:
+      - checkout
+      - install_js
+      - run:
+          name: Tests real browsers
+          # Run a couple of times for a better sample.
+          # TODO: hack something together where we can compile once and run multiple times e.g. by abusing watchmode.
+          command: |
+            yarn test:karma:profile
+            yarn test:karma:profile
+            yarn test:karma:profile
+            yarn test:karma:profile
+            yarn test:karma:profile
```

Comment:
```suggestion
            # Running on chrome only since actual times are innaccurate anyway
            # The other reason is that browserstack allows little concurrency so it's likely that we're starving other runs.
            yarn test:karma:profile --browser chrome,chromeHeadless
            yarn test:karma:profile --browser chrome,chromeHeadless
            yarn test:karma:profile --browser chrome,chromeHeadless
            yarn test:karma:profile --browser chrome,chromeHeadless
            yarn test:karma:profile --browser chrome,chromeHeadless
```